samples/meetings-sidepanel/nodejs/README.md

```diff
@@ -30,14 +30,14 @@ This sample illustrates how to implement [Side Panel](https://docs.microsoft.com
     # determine node version
     node --version
     ```
-- [socket.io](https://www.npmjs.com/package/socket.io) to update data in real time.
       
  - [Ngrok](https://ngrok.com/download) (Only for devbox testing) Latest (any other tunneling      software       can also be used)
-    ```bash
 
+    ```bash
      # run ngrok locally
     ngrok http -host-header=localhost 3978
```

Comment:
Please verify the port numbers and project folder path once as you have mentioned two ports like 3978 & 3001